pyvista/plotting.py

```diff
@@ -912,6 +912,8 @@ def add_mesh(self, mesh, color=None, style=None, scalars=None,
                 raise TypeError('Invalid texture type ({})'.format(type(texture)))
             if mesh.GetPointData().GetTCoords() is None:
                 raise AssertionError('Input mesh does not have texture coordinates to support the texture.')
+            if mesh.repeat_texture:
```

Comment:
add `repeat_texture` keyword argument to this method instead of accessing it from the mesh